allure-web/src/components/testcase/TestcaseView.hbs

```diff
@@ -1,7 +1,9 @@
 <a class="pane__overlay" href="#testcase/{{uid}}"></a>
-<div class="pane__subtitle long-line line-ellipsis">
-    <span class="fa fa-files-o fullname__copy" data-tooltip="Copy to clipboard"></span>
-    <span class="fullname__body">{{fullName}}</span>
+{{#if (is-def fullName)}}
```

Comment:
You don't need `is-def` helper here. It is only needed to split `undefined` with `0` (for numbers, not strings):

```js
export default function isDef(variable) {
    return (variable || typeof variable === 'number');
}
```